src/actions/example_store_actions/get_example_message.js

```diff
@@ -6,12 +6,12 @@ import Router, { checkStatus, handleError } from '../../core/router.js';
 import { GET_EXAMPLE_MESSAGE } from '../../constants.js';
 
 Actions.register(GET_EXAMPLE_MESSAGE, payload => {
-  Axios(Router.request('GET', GET_EXAMPLE_MESSAGE, {
-    path_args: {id: payload.id}
-  }))
-  .then(checkStatus)
-  .then(response => {
-    ExampleStore.setExampleMessage(response.data.message);
-    Actions.finish(payload);
-  }).catch(handleError);
+  // Axios(Router.request('GET', GET_EXAMPLE_MESSAGE, {
+  //   path_args: {id: payload.id}
+  // }))
+  // .then(checkStatus)
+  // .then(response => {
+  //   ExampleStore.setExampleMessage(response.data.message);
+  //   Actions.finish(payload);
+  // }).catch(handleError);
```

Comment:
Same as above. Are these comments left in intentionally?

Reply:
Same... We can delete these in the next go around...